addons/website_form/views/website_form_templates.xml

```diff
@@ -1,137 +1,181 @@
-                                            <input id="contact1" type="text" class="form-control s_website_form_input" name="Name" required=""/>
+    <template id="contactus" name="Contact Us">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <div class="oe_structure">
+                    <section class="s_title parallax s_parallax_is_fixed bg-black-50 pt24 pb24" data-vcss="001" data-snippet="s_title" data-scroll-background-ratio="1">
+                        <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_banner_default_image'); background-position: 50% 0;" />
+                        <div class="o_we_bg_filter bg-black-50" />
+                        <div class="container">
+                            <h1>Contact us</h1>
+                        </div>
+                    </section>
+                </div>
+                <section class="s_text_block" data-snippet="s_text_block">
+                    <div class="container mt-4 mb-5">
+                        <div class="row">
+                            <div class="col-lg-8 col-xl-6 mr-lg-auto">
+                                <div class="oe_structure">
+                                    <section class="s_text_block" data-snippet="s_text_block">
+                                        <div class="container">
+                                            <div class="row">
+                                                <div class="col-lg-12">
+                                                    <p>
+                                                        Contact us about anything related to our company or services.<br />
+                                                        We'll do our best to get back to you as soon as possible.
+                                                    </p>
+                                                </div>
+                                            </div>
                                         </div>
-                                    </div>
+                                    </section>
                                 </div>
-                                <div class="form-group col-12 s_website_form_field s_website_form_custom" data-type="char" data-name="Field">
-                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact2">
-                                            <span class="s_website_form_label_content">Phone Number</span>
-                                        </label>
-                                        <div class="col-sm">
-                                            <input id="contact2" type="tel" class="form-control s_website_form_input" name="Phone"/>
-                                        </div>
+                                <t t-call="website_form.contactus_form" />
```

Comment:
@tsm-odoo Closed the PR for a reason? Anyway be careful that these t-call are exactly what we discussed and that you should remove as they would make the page dynamic and thus non editable